### ISSUE TITLE
[FEATURE] On-hover dropdown navbar

### ISSUE THREAD

**Issue by kabir0x23** [CryptoverseWeb3/CryptoverseWeb3.com#4]
### Detailed description

Add Learn nav-item in the navbar
- remove transaction history nav-item
- 
![image](https://user-images.githubusercontent.com/44284877/192979347-a7b7e12d-fd8e-4c27-b927-ae6a72d78118.png)

add learn nav-item
Example: 
![image](https://user-images.githubusercontent.com/44284877/192978924-91b93fd5-dcc2-4312-8110-c5f79538d426.png)

### Context

- On-hover navbar is the best thing we can do to  make good user-experience 

### Additional information

If you need any help, let us know below: 👇👇

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MasterIfeanyi** [comment]
Hi, I would love to be assigned this issue.

**Comment by kabir0x23** [comment]
> Hi, I would love to be assigned this issue.

Nice, I am assigning you this issue.

The first step is:

- Will you check out this repository first?
- https://github.com/CryptoverseWeb3/git-github-practice
 - You will get more green squares

**Comment by MasterIfeanyi** [comment]
Is there already a component called learn, or do I have to create one by myself.

**Comment by kabir0x23** [comment]
> Is there already a component called learn, or do I have to create one by myself.

We are using [tailwindcss](https://tailwindcss.com/) here.
- you can visit [tailwindcss](https://tailwindcss.com/) website.

if you need any help, let us know.

**Comment by MasterIfeanyi** [comment]
No, I mean do I have to create a react component called "Learn.jsx" to add to the nav-item.

**Comment by kabir0x23** [comment]
> No, I mean do I have to create a react component called "Learn.jsx" to add to the nav-item.

oh okay.
- no, there is no component called learn and there is no need to create a new component. 
- just add nav-item called learn.
 - and on hover on that learn nav-item, 2-3 subitems should be displayed 
 - examples: 
 - Learn:
   - > blogs
   - > crypto basics
   - > video tutorials

**Comment by MasterIfeanyi** [comment]
This was a very great explaination. Your explaination right now should be made Industry standard. cool, assign this issue to me. Please.

**Comment by kabir0x23** [comment]
> This was a very great explaination. Your explaination right now should be made Industry standard. cool, assign this issue to me. Please.

Thanks, @MasterIfeanyi for the kind words.
This issue has been assigned to you.

**Comment by faisalamin001** [comment]
I am available and would love to add the "Learn" Nav-item, if it hasn't been added yet
Thanks
